packages/interceptors-opentelemetry/src/workflow/index.ts

```diff
@@ -1,33 +1,45 @@
 import './runtime'; // Patch the Workflow isolate runtime for opentelemetry
 import * as otel from '@opentelemetry/api';
-import * as tracing from '@opentelemetry/sdk-trace-base';
 import {
   ActivityInput,
   Next,
   WorkflowInboundCallsInterceptor,
   WorkflowOutboundCallsInterceptor,
   WorkflowExecuteInput,
 } from '@temporalio/workflow';
-import { defaultDataConverter } from '@temporalio/common';
+import { extractSpanContextFromHeaders, insertSpanContextIntoHeaders } from '@temporalio/common';
 import { instrument, instrumentFromSpanContext } from '../instrumentation';
 import { ContextManager } from './context-manager';
 import { SpanExporter } from './span-exporter';
 import { SpanName } from './interfaces';
 
 export * from './interfaces';
 
-export const TRACE_HEADER = 'Otel-Trace-Context';
-
-export const tracer = otel.trace.getTracer('workflow');
-
+// TODO: Literally the existence of this function causes us to blow up
+//   trying to read Date.now() too early (not calling it, just it being defined)
+//   I think the problem is importing tracing literally calls Date.now
```

Comment:
@bergundy What do you think we can do about initializing Date.now() earlier?

I actually prefer my lazy-initted tracer to this anyway, but the problem remains that the runtime fixer-upper thing doesn't work right b/c Date.now() can't be called at import time.

Reply:
This is a regression, Date should be available at import time.

Reply:
OK what do we need to do to get it initted properly?